AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1282,6 +1282,7 @@ slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,itprotoday.com
 bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.com,swindonadvertiser.co.uk,wiltshiretimes.co.uk##.cta-component-full-width
 !
 lcsun-news.com##gannett-atoms-component-newsletter-cta[link-newsletter]
+fashionwaltz.com#?##sidebar-2 > div.widget:has(> div.textwidget > div#newsletter_form)
```

Comment:
I guess you can exchange this part`#sidebar-2` to `.pull-left`. Also for social ext-css rule.